### DISCUSSION TITLE
Dead code that blocks Docker build

### DISCUSSION THREAD

**Discussion by SofiyanIfren** [payloadcms/payload#8912] [Q&A]
### Describe the Bug

Hi !  Thank you for your amazing job ;)  

When deploying via Dockerfile, we found an issue with my team : there is some code that doesn't work and stops the build, at line 22 in /src/payload.config.ts :  

`import CustomNavbar from './component/CustomNavBar'` 

This component isn't used and... is inexisting, so I just removed it and it worked well.  

I can make a PR if you want ;)  H

ave a nice day

### To Reproduce

Just run docker build -t cms-proviridis:1.1.1 .

### Payload Version

    "@payloadcms/bundler-webpack": "^1.0.0",     "@payloadcms/db-mongodb": "^1.0.0",     "@payloadcms/plugin-cloud": "^3.0.0",     "@payloadcms/richtext-slate": "^1.0.0",

**Comment by jmikrut** [comment]
Hi, I am not sure what you're referring to. Are you talking about a Payload template? Where is the `CustomNavBar` import?

We would happily merge a PR if I knew what this was in reference to!

**Comment by SofiyanIfren** [comment]
Yes, here is some context : while working on the view (we wanted to modify some wording in the title), we made this import on the screenshot :

![image](https://github.com/user-attachments/assets/f46913a5-c5a2-46b3-b6d3-2553fefbd960)

Then, when building with docker, this unused import created an error
